src/file-uploader/file-uploader.stories.ts

```diff
@@ -18,6 +18,8 @@ import {
 } from "../";
 import { NotificationService } from "../notification/notification.service";
 
+const fileType = require("file-type");
```

Comment:
Can we `import { fileType } from "file-type";` or `import * as fileType from "file-type";`?